File: Sheet_1_Solutions/Day_4_5/Q1.cpp
// Write a Program to Check Whether Number is Even or Odd


#include <iostream> // Include standard library and namespace
using namespace std; // for C++

int main(){
    int n;
    cout<<"Enter a number: ";
    cin >> n;

    if (n%2==0){ // Check if the no. is perfectly divisible by two or not.

        cout<<"Entered number is a even no.";
    
    } else {
    
        cout<<"Entered number is a odd no.";
    
    }

}

File: Sheet_1_Solutions/Java/Day 4&5/Q1.java
import java.util.Scanner;

public class Q1 {

	public static void main(String[] args) {
	Scanner ss = new Scanner(System.in);
	System.out.print("Enter a number to cheak it is even or odd: ");
	int num = ss.nextInt();
	if(num%2==0)
	System.out.println("Number is EVEN");
	else
	System.out.println("Number is ODD");
    }
}